test/test_mps.py

```diff
@@ -5716,25 +5716,7 @@ def run_test(device, op):
             op = partial(fn, source=0, destination=1)
             run_test(device, op)
 
-    # Testing that the generated view_copy kernel and its derivative are implemented correctly
-    def test_view_copy(self, device="mps"):
```

Comment:
Do OpInfos run MPS tests? cc @kulinseth @albanD

Reply:
Yes we have test to ensure that CPU match with MPS.